README.md

```diff
@@ -135,6 +135,39 @@ function will not load any piece of the imaging data into memory until you speci
 call `.compute` on the returned Dask array. In doing so, you will only then load the
 selected chunk in-memory.
 
+### Mosaic Image Reading
+
+Read stitched data or single tiles as a dimension.
+
+Readers that support mosaic tile stitching:
+
+-   `LifReader`
+
+#### AICSImage
+
+If the file format reader supports stitching mosaic tiles together, the
+`AICSImage` object will use the stitched image data.
+
+```python
+img = AICSImage("very-large-mosaic.lif")
+img.dims.order  # T, C, Z, big Y, big X, (S optional)
+img.dask_data  # Dask chunks fall on tile boundaries, pull YX chunks out of the image
+```
+
+#### Reader
+
+If the file format reader detects mosaic tiles in the image, the `Reader` object
+will store the tiles as a dimension.
+
+If implemented, the `Reader` can also return the stitched image.
+
+```python
+reader = LifReader("ver-large-mosaic.lif")
+reader.dims.order  # M, T, C, Z, tile size Y, tile size X, (S optional)
+reader.dask_data  # normal operations, can use M dimension to select individual tiles
+reader.mosaic_dask_data  # returns stitched mosaic - T, C, Z, big Y, big, X, (S optional)
```

Comment:
This isn't quite clear to me: from reading this doc, is M now a standard dimension in aicsimageio?  

If i call reader.dask_data, am I NOT getting the reconstructed full image?  What if I omit "M" in my args to reader.dask_data?

If I call aicsimage.dask_data, am I getting the reconstructed full image?  Will my dask YX chunks have different dims than dims.YX ?

Reply:
Just a thought - maybe if the example code had some numbers for YXreconstructed = (2048, 4096) and YXtile = (512, 1024) then it would be clearer what was being requested vs what gets returned.